packages/algolia-experiences/package.json

```diff
@@ -10,7 +10,7 @@
   ],
   "dependencies": {
     "algoliasearch": "5.1.1",
-    "instantsearch.js": "4.75.3"
+    "instantsearch-core": "0.1.0"
```

Comment:
We're importing all the widgets from instantsearch.js in src/widgets.ts, does this change work?

Reply:
🤦 yes needs to be changed indeed, I missed that import